CMGTools/TTHAnalysis/python/analyzers/ttHReclusterJetsAnalyzer.py

```diff
@@ -12,14 +16,14 @@ def declareHandles(self):
        #genJets                                                                                                                                                                     
         self.handles['genJets'] = AutoHandle( 'slimmedGenJets','std::vector<reco::GenJet>')
 
-    def beginLoop(self):
-        super(ttHReclusterJetsAnalyzer,self).beginLoop()
+    def beginLoop(self, setup):
+        super(ttHReclusterJetsAnalyzer,self).beginLoop(setup)
         self.counters.addCounter('pairs')
         count = self.counters.counter('pairs')
         count.register('all events')
 
     def makeFatJets(self, event):
-        objects40jc = [ j for j in event.cleanJets if j.pt() > 40.0 and abs(j.eta())<2.4 ]
+        objects40jc = [ j for j in event.cleanJets if j.pt() > 30.0 and abs(j.eta())< 5.0 ]
```

Comment:
while it's technically correct, maybe it would be cleaner not to call "objects40jc" jets that are 30 GeV and no longer central.
Also, for this case I think the cleanest solution is to start from `event.cleanJetsAll` make the p<sub>T</sub> and |&eta;| cuts configurable in the `ttHReclusterJetsAnalyzer`.
Could you do it and update the PR?